### PR TITLE
gravel: node/mgr: share ceph.conf and keyring on join

### LINKED ISSUE #314: gravel: need to pass ceph.conf et al during node join
Once a node fully joins the cluster, it needs a ceph.conf and a keyring to be able to perform operations. At the moment these are not being passed to it during the join worflow, and that needs to change.

### PR DESCRIPTION
**Pull request by jecluis**
Resolves #314

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Review by mgfritch — APPROVED** [review]

File: src/gravel/controllers/nodes/mgr.py [hunk 1]
```diff
@@ -271,6 +273,13 @@ async def join(self, leader_address: str, token: str) -> bool:
             fd.writelines([welcome.pubkey])
             logger.debug(f"join > wrote pubkey to {authorized_keys}")
 
+        cephconf_path: Path = Path("/etc/ceph/ceph.conf")
+        keyring_path: Path = Path("/etc/ceph/ceph.client.admin.keyring")
+        if not cephconf_path.parent.exists():
+            cephconf_path.parent.mkdir(0o755)
+        cephconf_path.write_text(welcome.cephconf)
+        keyring_path.write_text(welcome.keyring)
```
[the remark this answers is not in the document]
Comment (jecluis) [reply src/gravel/controllers/nodes/mgr.py:281]: although chown'ed already by `root:root`

**Comment by jecluis** [comment]
Jenkin failures are flukes.

**Comment by jecluis** [comment]
And npm tests are irrelevant for this patch. And seem to be failing on install of packages.

**Comment by jecluis** [comment]
@mgfritch mind re-approving?

**Comment by jecluis** [comment]
jenkins run tumbleweed

**Review by mgfritch — APPROVED** [review]

**Pull request merged**